docs/notebooks/getting_started/particles.ipynb

```diff
@@ -0,0 +1,903 @@
+   "outputs": [],
+   "source": [
+    "water = molecule(\"H2O\")\n",
+    "water.symbol"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff5a6cf1",
+   "metadata": {},
+   "source": [
+    "The properties of the molecule are found automatically."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ae6284a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "water.mass"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "23c42034",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "acetic_acid_anion = molecule(\"CH3COOH 1-\")\n",
+    "acetic_acid_anion.charge"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebc037f7",
+   "metadata": {},
+   "source": [
+    "## Particle categorization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f298538f",
+   "metadata": {},
+   "source": [
+    "[categories]: ../../api/plasmapy.particles.particle_class.Particle.rst#plasmapy.particles.particle_class.Particle.categories\n",
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "\n",
+    "The [categories] attribute of a [Particle] provides a set of the categories that the [Particle] belongs to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "09f122ff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "muon = Particle(\"muon\")\n",
+    "muon.categories"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "934739a8",
+   "metadata": {},
+   "source": [
+    "[is_category()]: ../../api/plasmapy.particles.particle_class.Particle.rst#plasmapy.particles.particle_class.Particle.is_category\n",
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "\n",
+    "The [is_category()] method lets us determine if a [Particle] belongs to one or more categories."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d278fa15",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "muon.is_category(\"lepton\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "10c5ce7c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proton.is_category(\"boson\", \"baryon\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6edf5cc1",
+   "metadata": {},
+   "source": [
+    "[Particle]: ../../api/plasmapy.particles.particle_class.Particle.rst\n",
+    "\n",
+    "If we need to be more specific, we can use the `require` keyword for categories that a [Particle] must belong to, the `exclude` keyword for categories that the [Particle] cannot belong to, and the `any_of` keyword for categories of which a [Particle] needs to belong to at least one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5b5e6ee7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "electron.is_category(require=\"lepton\", exclude=\"baryon\", any_of={\"boson\", \"fermion\"})"
```

Comment:
Reference to [issue 1521](https://github.com/PlasmaPy/PlasmaPy/issues/1521) :  having this functionality expanded and work on `ParticleList` objects would be a good future project.

Reply:
I had forgotten about it, but there is such a thing as a ~~tesseract~~ [`ParticleList.is_category`](https://docs.plasmapy.org/en/latest/api/plasmapy.particles.particle_collections.ParticleList.html#plasmapy.particles.particle_collections.ParticleList.is_category)!  Since you mentioned it, I should add something about that here too.